recipes/python-duckdb/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "duckdb" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: python-{{ name|lower }}
```

Comment:
@chrisburr I changed the name like, do you think this is ok or do we need the `name` variable to match the actual package name?

Reply:
The jinja2 variable doesn't matter so it's fine like this.

Have you tried installing with pip like the linter now suggests?

Reply:
Yes, and I've raised https://github.com/cwida/duckdb/issues/227 upstream.

Reply:
Great, LGTM 🎉